### PR TITLE
[Doc] add casdoor auth provider

### PR DESCRIPTION
**Pull request by akriventsev**

### CODE REVIEW

**Review by djhi — CHANGES_REQUESTED** [review]

File: docs/AuthProviderList.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ It's very common that your auth logic is so specific that you'll need to write y
 - **[Postgrest](https://postgrest.org/)**: [raphiniert-com/ra-data-postgrest](https://github.com/raphiniert-com/ra-data-postgrest)
 - **[Supabase](https://supabase.io/)**: [marmelab/ra-supabase](https://github.com/marmelab/ra-supabase)
 - **[SurrealDB](https://surrealdb.com/)**: [djedi23/ra-surrealdb](https://github.com/djedi23/ra-surrealdb)
+- **[Casdoor](https://casdoor.com/)**: [NMB-Lab/reactadmin-casdoor-authprovider](https://github.com/NMB-Lab/reactadmin-casdoor-authprovider)
```
[the remark this answers is not in the document]
Comment (akriventsev) [reply docs/AuthProviderList.md]: Done

File: docs/AuthProviderList.md [hunk 2]
```diff
@@ -11,10 +11,11 @@ It's very common that your auth logic is so specific that you'll need to write y
...
 - **[Azure Active Directory (using MSAL)](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-browser)**: [marmelab/ra-auth-msal](https://github.com/marmelab/ra-auth-msal)
+- **[Casdoor](https://casdoor.com/)**: [NMB-Lab/reactadmin-casdoor-authprovider](https://github.com/NMB-Lab/reactadmin-casdoor-authprovider)
 - **[Directus](https://directus.io/)**: [marmelab/ra-directus](https://github.com/marmelab/ra-directus)
 - **[Firebase Auth (Google, Facebook, GitHub, etc.)](https://firebase.google.com/docs/auth/web/firebaseui)**: [benwinding/react-admin-firebase](https://github.com/benwinding/react-admin-firebase#auth-provider)
 - **[Keycloak](https://www.keycloak.org/)**: [marmelab/ra-keycloak](https://github.com/marmelab/ra-keycloak)
-- **[Postgrest](https://postgrest.org/)**: [raphiniert-com/ra-data-postgrest](https://github.com/raphiniert-com/ra-data-postgrest)
+- **[Postgres](https://postgrest.org/)**: [raphiniert-com/ra-data-postgrest](https://github.com/raphiniert-com/ra-data-postgrest)
```
Comment (djhi) [inline docs/AuthProviderList.md:18]: Please revert this change. It is **Postgrest**

**Review by djhi — CHANGES_REQUESTED** [review]

**Review by djhi — APPROVED** [review]
Thanks!
